winui/Rating/Placeholder-value.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
```

Comment:
Layout: Post

Reply:
We have follow the small letter for all other controls.